_includes/prod_deployment/synchronize-clocks.md

```diff
@@ -0,0 +1,145 @@
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo timedatectl set-ntp no
+    ~~~
+
+    Verify that `timesyncd` is off:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ timedatectl
+    ~~~
+
+    Look for `Network time on: no` or `NTP enabled: no` in the output.
+
+3. Install NTP:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo apt-get install ntp
+    ~~~
+
+4. Force the machine's clock to synch with Google's NTP service:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo service ntp stop
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo ntpd -b time.google.com
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo service ntp start
+    ~~~
+
+    {{site.data.alerts.callout_info}}We recommend Google's NTP service because they handle <a href="https://developers.google.com/time/smear">"smearing" the leap second</a>. If you use an external NTP service that doesn't smear the leap second, be sure to configure client-side smearing in the same way on each machine.{{site.data.alerts.end}}
+
+5. Verify that the machine is using a Google NTP server:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo ntpq -p
+    ~~~
+
+    The active NTP server will be marked with an asterisk.
+
+4. Repeat these steps for each machine where a CockroachDB node will run.
+
+{% elsif page.title contains "Google" %}
+
+Compute Engine instances are preconfigured to use [NTP](http://www.ntp.org/), which should keep offsets in the single-digit milliseconds. However, Google can’t predict how external NTP services, such as `pool.ntp.org`, will handle the leap second. Therefore, you should:
+
+- [Configure each GCE instances to use the Google's internal NTP service](https://cloud.google.com/compute/docs/instances/managing-instances#configure_ntp_for_your_instances).
+- If you plan to run your cluster across cloud providers, [configure the non-GCE machines to use Google's external NTP service](deploy-cockroachdb-on-digital-ocean.html#step-2-sychronize-clocks).
+
+{% elsif page.title contains "AWS" %}
+
+Amazon provides the [Amazon Time Sync Service](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/set-time.html), which uses a fleet of satellite-connected and atomic reference clocks in each AWS Region to deliver accurate current time readings. The service also smears the leap second.
+
+- [Configure each AWS instance to use the internal Amazon Time Sync Service](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/set-time.html#configure-amazon-time-service).
+- If you plan to run your cluster across cloud providers, [configure the non-AWS machines to use Google's external NTP service](deploy-cockroachdb-on-digital-ocean.html#step-2-sychronize-clocks), which is comparably accurate and also handles <a href="https://developers.google.com/time/smear">"smearing" the leap second</a>.
```

Comment:
Does amazon not offer an externally-available time server that we can recommended for non-AWS instances?